ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/ProjectService.cs

```diff
@@ -80,5 +80,11 @@ public async Task<ProjectDTO> UpdateAsync(ProjectDTO projectDto, CancellationTok
             await _repository.UpdateAsync(project, cancellationToken);
             return projectDto;
         }
+
+        public Task<bool> ActivityExistsAsync(CreateTaskDTO task, CancellationToken cancellationToken)
```

Comment:
You shouldn't pass task dto to project service, task is not responsibility of project service. See the name of your method and pass only necessary info, the method should be universal and can be called from each place of app, doing necessary thing: checking if activity is in project